app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -116,6 +116,13 @@ $search-icon-height: 20px;
   padding-top: 16px;
   flex-grow: 1;
   flex-shrink: 0;
+
+  // govuk-frontend includes spacing to the right of the logo to offset it from the product name
+  // as we do not include the product name in the super navigation menu we can remove this to make best use
+  // of the space available at smaller screen sizes
+  .govuk-header__link--homepage {
+    margin-right: 0;
+  }
 }
```

Comment:
This override might not be needed. I tested on a 320px screen, and everything still fits even with the style disabled. Below 320px it does give a little extra room (down to about 310px), but the reflow criterion only requires us to avoid reflow at 320px.